src/index.js

```diff
@@ -83,6 +82,10 @@ function getUserLanguage () {
   )
 }
 
+function canReadClipboard () {
+  return !!navigator.clipboard && !!navigator.clipboard.readText
```

Comment:
😞 javascript

Reply:
Yep, this was pretty frustrating, but oh well :cry: